packages/api/src/routes/medical/patient.ts

```diff
@@ -214,6 +214,30 @@ router.get(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /patient/:id/consolidated/query/:requestId
+ *
+ * Returns the status and information on a specific data query for a given patient.
+ *
+ * @param req.param.id The ID of the patient whose data is to be returned.
+ * @param req.param.requestId The ID of the query to be returned.
+ * @returns the status and information on a specific data query for a given patient.
```

Comment:
nit: I know the endpoint above says "data" but that's in the context of "consolidated data"... I think we either keep using "consolidated data" or just use "consolidated". Just using "data" brings a question as to what data?

Reply:
yep good point, thats a weird doc string. Should probably be "... whose consolidated query status is to be returned." no?

Reply:
I think if we remove "...for a given patient" it should do, since those are for a specific request ID.

Reply:
I think the route makes sense. I see the argument for not having the patientId on the route, but also, when we trigger a consolidated query we need the patient ID there, to the route for the individual status need to follow that pattern. Also, the consolidated query doesn't exist by itself on the system, its also tied to a patient.